conf/evolutions/default/221.sql

```diff
@@ -0,0 +1,13 @@
+# --- !Ups
+CREATE TABLE panorama_history (
+   panorama_id TEXT NOT NULL,
```

Comment:
Let's call this either `pano_id`, bc that's what we use most of the time. The other option is the more verbose `gsv_panorama_id`, but I don't see a reason to use that. GSVs are the only panos we have :)